website/docs/internals/provider-registry-protocol.mdx

```diff
@@ -46,7 +46,7 @@ is optional, and if omitted defaults to `registry.opentofu.org/`.
 For example:
 
 * `hashicorp/aws` is a shorthand for `registry.opentofu.org/hashicorp/aws`,
-  which is the official AWS provider published by HashiCorp.
+  which is the official AWS provider published by OpenTofu.
```

Comment:
This is actually incorrect, I think we should keep this as hashicorp.
```suggestion
  which is the official AWS provider published by HashiCorp.
```

Reply:
Sorry for that, got a bit carried away with replacements 😄

Reply:
Uh, You're not the author of this PR @kislerdm ? I'm confused...

Reply:
@Yantrio I suggested it in my [comment](https://github.com/opentofu/opentofu/pull/902#discussion_r1400334174), so it's partially my responsibility 😄